README.md

```diff
@@ -86,10 +86,20 @@ paramount**. `simpleCache` assumes that your name for an object is a perfect
 identifier for that object; in other words, don't cache things that you plan to
 change.
 
+### Highlights of exported functions
+
+- `simpleCache()`: Caches and reloads objects created based on instruction
+- `listCaches()`: Liosts all of the caches available in the `cacheDir`
+- `deleteCaches()`: Deletes cache(s) from the `cacheDir`
+- `setCacheDir()`: Sets a global option for a cache directory so you don't have to specify one in each `simpleCache` call
+- `viewCacheDirs()`: Views all of the `simpleCache` global options that have been set
```

Comment:
Does this function really do that? If so it needs to be renamed... or is this wrong?